sql/plan/show_create_table_test.go

```diff
@@ -33,13 +33,13 @@ func TestShowCreateTable(t *testing.T) {
 
 	table := memory.NewTable(
 		"test-table",
-		sql.Schema{
+		sql.NewPrimaryKeySchema(sql.Schema{
 			&sql.Column{Name: "baz", Type: sql.Text, Default: nil, Nullable: false, PrimaryKey: true},
 			&sql.Column{Name: "zab", Type: sql.Int32, Default: parse.MustStringToColumnDefaultValue(ctx, "0", sql.Int32, true), Nullable: true, PrimaryKey: true},
 			&sql.Column{Name: "bza", Type: sql.Uint64, Default: parse.MustStringToColumnDefaultValue(ctx, "0", sql.Uint64, true), Nullable: true, Comment: "hello"},
 			&sql.Column{Name: "foo", Type: sql.MustCreateStringWithDefaults(sqltypes.VarChar, 123), Default: nil, Nullable: true},
 			&sql.Column{Name: "pok", Type: sql.MustCreateStringWithDefaults(sqltypes.Char, 123), Default: nil, Nullable: true},
-		})
+		}, []int{1}))
```

Comment:
This doesn't match the above

Reply:
Probably similar errors in other test files, I have not checked exhaustively

Reply:
i'll do another lookthrough to try and catch stragglers